packages/client-adapters/src/adapters/lexicons/adapter.js

```diff
@@ -43,15 +44,15 @@ class AlpheiosLexiconsAdapter extends BaseAdapter {
   * @param {String} urlKey - urlIndex for geting data from config
   */
   prepareShortDefPromise (homonym, urlKey) {
-    const url = this.config[urlKey].urls.short
+    const url = this.config.lexicons[urlKey].urls.short

```

Comment:
@balmas , may be it is better to save to the constructor only appropriate config - this.config = remoteConfig.lexicons ?
For better consistency? 
I believe that other parts of config (for example for logeion should be passed to logeon?

Reply:
I'm not sure I understand your suggestion exactly.  I am using "lexicons" here to separate out the adapter level config (e.g. methods, etc.) from the content level config.  The full remote config (i.e. with logeion etc.) is not passed to the lexicons adapter, only the lexicons piece.

Reply:
Oh, I see, I mixed lexicons property in the _appConfig and in the local config of ClientAdapters.
I am sorry.
Config for ClientAdapters becomes bigger and bigger :)